### PR TITLE
feat : solve step 2.

### PR DESCRIPTION
**Pull request by jukqaz**
solve baekjoon step 2.

### CODE REVIEW

File: 2/hmh/1_1000.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package `2`.hmh
+
+import java.util.*
+
+object Example2_1 {
+	@JvmStatic
+	fun main(args: Array<String>) {
```
Comment (jukqaz) [inline 2/hmh/1_1000.kt]: a

**Review by AlvinYou — APPROVED** [review]
👍

**Review by korJiusngPark — APPROVED** [review]
good

**Review by kor-haru — APPROVED** [review]
Good

**Review by cjy9306 — APPROVED** [review]
goood

**Pull request merged**